Refined Python Docker image and updated script to run all tests (#4489)
awsdocs/aws-doc-sdk-examples: python/README.md

python -m pytest -m "integ"
```

## Docker image

This example code will soon be available in a container image

> hosted on [Amazon Elastic Container Registry (ECR)](https://docs.aws.amazon.com/AmazonECR/latest/userguide/what-is-ecr.html). 
> This image will be pre-loaded with all Python examples with dependencies pre-resolved, 
> allowing you to explore these examples in an isolated environment.

Reviewer: ```suggestion
hosted on [Amazon Elastic Container Registry (Amazon ECR)](https://docs.aws.amazon.com/AmazonECR/latest/userguide/what-is-ecr.html). 
The image will be preloaded with all Python examples, with dependencies pre-resolved. 
That way, you can explore the examples in an isolated environment.
```

Author: fixed
